crates/cheatnet/src/runtime_extensions/forge_runtime_extension/mod.rs

```diff
@@ -496,6 +494,20 @@ impl<'a> ExtensionLogic for ForgeExtension<'a> {
                     ]))
                 }
             }
+            "get_last_transaction_trace" => {
```

Comment:
Actually, why constrain it to latest call?

Reply:
@drknzz the name may be misleading, it is the last call from the perspective of the test code. I decided to constraint it to make sure the UX is better, otherwise `get_last_call_trace` would return the trace from every call happened from the beginning of the test, it would be unreadable

Reply:
Yes, I do understand what it does currently. Just asking why not allow to get every trace, but just the last one? I think some enum parameter wouldn't make the UX that much worse

Reply:
Actually it could be implemented, then we could have the two functions exposed to users - `get_last_call_trace` and `get_full_trace`, the latter would return `Array<Array<CallEntryPoint>>`

Reply:
It's a valid concern i think, can we make an issue?

Reply:
#1436